### PR TITLE
refactor factories

### PR DESCRIPTION
**Pull request by faissaloux**

### CODE REVIEW

File: database/factories/AcceptanceFactory.php [hunk 1]
```diff
@@ -11,12 +11,8 @@ class AcceptanceFactory extends Factory
...
     {
         return [
-            'talk_revision_id' => function () {
-                return TalkRevision::factory()->create()->id;
-            },
-            'conference_id' => function () {
-                return Conference::factory()->create()->id;
-            },
+            'talk_revision_id' => TalkRevision::factory()->create(),
+            'conference_id' => Conference::factory()->create(),
```
Comment (andrewmile) [inline database/factories/AcceptanceFactory.php:15]: You should be able to drop the `->create()` from these lines.

**Review by andrewmile — CHANGES_REQUESTED** [review]
Thanks @faissaloux , Can you remove the `create` chaining from the end of these factory calls? After that I think it will be good to go.

**Comment by faissaloux** [comment]
@andrewmile 👀 Done!

**Review by andrewmile — APPROVED** [review]
Perfect, thank you!

**Comment by faissaloux** [comment]
Happy to help! 😄

**Pull request merged**